api/src/lib.rs

```diff
@@ -56,3 +56,9 @@ pub use crate::rest::*;
 pub use crate::router::*;
 pub use crate::types::*;
 pub use crate::web::*;
+
+/// Get max transaction size
+pub fn max_tx_size() -> u64 {
+	p2p::msg::max_block_size()
+		- std::mem::size_of_val(&core::core::block::BlockHeader::default()) as u64
```

Comment:
This does not necessarily correspond to a serialized size in bytes. Its simply the internal representation, which will likely be different.